docs/README.md

```diff
@@ -0,0 +1,3 @@
+# Documentation
```

Comment:
Is this expected to actually show in the Documentation groups list too? Seems a bit meta if so!

Reply:
Yep, it's a module that you get included, so makes sense to be documented (similar to Core).